core/src/FieldLabel/FieldLabel.less

```diff
@@ -0,0 +1,38 @@
+@import "../colors.less";
+@import "../typography.less";
+
+.TextFieldTypeLabel {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+  font-size: @FieldLabelSize;
```

Comment:
The next few lines contain camel case, snake case and hyphenated variables. Lets pick one and stick with it. Again changing these can be breaking for any consumers. That is okay we just have to be aware of it and update the consuming apps.

Reply:
Good point. I fixed the casing. Hyphenated variables are from the old color LESS file, these variable are consuming in a lot of places. Hyphenated matches CSS naming conventions. I'm ok with picking a hyphenated across the board if you are. Let me know your thoughts.

Are any of our apps consuming the colors.less or typography.less files individually?

Reply:
I updated these to have legacy variables so it will not error externally.